azure-kusto-ingest/azure/kusto/ingest/exceptions.py

```diff
@@ -1,39 +1,39 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License
-
+from azure.kusto.data.data_format import DataFormat, IngestionMappingKind
 from azure.kusto.data.exceptions import KustoClientError
 
 
-class KustoMappingAndMappingReferenceError(KustoClientError):
+class KustoMappingError(KustoClientError):
```

Comment:
What's the benefit of another layer in the hierarchy?
I'm not sure it adds value, especially when we're trying to move away from most mapping errors (including `KustoMissingMappingError`)

Reply:
I needed an exception to throw that is related to Mapping, but didn't fit any of the ones we had so I created the general "KustoMappingError", and from there if I was a user, I'd expect that catching a "KustoMappingError" will catch every type of mapping error, so hence the inhertience.

Reply:
But there are only 2. is it worth adding a hierarchy to group together 2 exceptions that don't share any properties, but instead share 1 common theme?
Also, 1 of those 2 will be disappearing soon.

Is it really surprising to a user to get raised a `KustoDuplicateMappingError(KustoClientError)` or a `KustoMissingMappingReferenceError(KustoClientError)`?

If we look at https://www.tutorialspoint.com/java/java_builtin_exceptions.htm we see many exceptions (e.g. `ArrayIndexOutOfBoundsException`, `ArrayStoreException`, `NegativeArraySizeException` all relating to arrays) that share a common theme, but since they don't share uniquely common members, they don't have an OO hierarchy.

Reply:
Removed hierarchy